lms/static/js/learner_dashboard/views/course_enroll_view_2017.js

```diff
@@ -0,0 +1,111 @@
+(function(define) {
+    'use strict';
+
+    define(['backbone',
+        'jquery',
+        'underscore',
+        'gettext',
+        'edx-ui-toolkit/js/utils/html-utils',
+        'text!../../../templates/learner_dashboard/course_enroll_2017.underscore'
+    ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             HtmlUtils,
+             pageTpl
+         ) {
+             return Backbone.View.extend({
+                 className: 'course-enroll-view',
+
+                 tpl: HtmlUtils.template(pageTpl),
+
+                 events: {
+                     'click .enroll-button': 'handleEnroll',
+                     'change .run-select': 'handleCourseRunSelect'
+                 },
+
+                 initialize: function(options) {
+                     this.$parentEl = options.$parentEl;
+                     this.enrollModel = options.enrollModel;
+                     this.urlModel = options.urlModel;
+                     this.render();
+                     if (this.urlModel) {
+                         this.trackSelectionUrl = this.urlModel.get('track_selection_url');
+                     }
+                 },
+
+                 render: function() {
+                     var filledTemplate;
+                     if (this.$parentEl && this.enrollModel) {
+                         filledTemplate = this.tpl(this.model.toJSON());
+                         HtmlUtils.setHtml(this.$el, filledTemplate);
+                         HtmlUtils.setHtml(this.$parentEl, HtmlUtils.HTML(this.$el));
+                     }
+                 },
+
+                 handleEnroll: function() {
+                    // Enrollment click event handled here
+                     if (!this.model.get('course_run_key')) {
+                         this.$('.select-error').css('visibility', 'visible');
```

Comment:
I would prefer to use a CSS class that's added or removed instead of setting the element's style directly.